hrui/js/src/civihr-popup/civihr-popup.js

```diff
@@ -4,14 +4,61 @@ Prevent Popups to overflow tables.
 (function ($) {
   'use strict';
 
-  // .btn-slide is a continer of popup and a button triggering it
-  $('.btn-slide').attrchange(function (e) {
-    var $body = $('body');
+  $('body').on('click', '.btn-slide', function (e) {
     var $button = $(this);
-    var $closestEntity, $popup, buttonOffset;
+    var $body = $('body');
+    var $popup, buttonOffset, $closestEntity;
+
+    e.preventDefault();
+
+    (function init () {
+      openPopupPanel();
+      listenToMouseOutEvent();
+    })();
+
+    /**
+     * Closes the popup panel and removes the .civihr-popup-open class from the
```

Comment:
I think it would be easier to read if you break the sentence like that:
```js
/**
 * Closes the popup panel
 * removes the .civihr-popup-open class from the body
 */
```